.rubocop.yml

```diff
@@ -109,6 +114,9 @@ Style/TernaryParentheses:
 Style/IfUnlessModifier:
   Enabled: false
 
+Style/RedundantBegin:
+  Enabled: false
```

Comment:
Why disable this?

Reply:
This file mimics caseflow repo. As you can see:
https://github.com/department-of-veterans-affairs/caseflow/blob/master/.rubocop.yml#L21

Reply:
I will change this as well. Let me know if you spot anything else.